test/e2e/common_test.go

```diff
@@ -3,12 +3,16 @@ package integration
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"sort"
 	"strings"
 	"testing"
 
+	"github.com/containers/storage"
+
 	"github.com/containers/libpod/pkg/inspect"
 	. "github.com/containers/libpod/test/utils"
 	"github.com/containers/storage/pkg/reexec"
```

Comment:
Probably my lack of knowledge: I don't see `storage.` or `rexec.` referenced anywhere in here.  Yet this PR passed the validation stuffs - which I'm assuming catch orphaned imports, no?

Reply:
lines 315 and 75 respectively

Reply:
OIC it now, thx for reference.